flatpak/org.feichtmeier.Musicpod.appdata.xml

```diff
@@ -14,7 +14,9 @@
       Play local audio files, browse podcasts online or listen to internet radio stations.
     </p>
   </description>
-  
+  <developer>
```

Comment:
needs id https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#name-summary-and-developer-name

Reply:
Luckily, Ubuntu Flutter Community owns ubuntufluttercommunity.org, so I used that. I'm assuming that'll work, right?

Reply:
Yea anything would work, neither Appstream nor Flathub checks the validity of the id here.